pkg/dock/discovery/discovery.go

```diff
@@ -21,6 +21,8 @@ package discovery
 
 import (
 	"fmt"
+	"github.com/opensds/opensds/contrib/drivers/utils/config"
```

Comment:
Please use `make goimports`  command to format the code style.

Reply:
Done